test/stdlib/Character.swift

```diff
@@ -310,8 +310,8 @@ CharacterTests.test(
   expectEqualMethodsForDomain(
     ascii0to126,
     ascii1to127,
-    { x in { String(x) < String($0) } } as PredicateFn,
-    { x in { String(Character(x)) < String(Character($0)) } } as PredicateFn)
```

Comment:
Nit: if `PredicateFn` is no longer useful, `typealias` could be removed as well..

Reply:
D'oh! Yes, will push another commit to remove that.